README.md

```diff
@@ -16,7 +16,7 @@ For more information please check:
 
 - [OpenEuropa components](docs/openeuropa-components.md)
 - [How to contribute](docs/how-to-contribute.md)
-
+- [Drupal Core experimental module policy](docs/experimental-module-policy.md)
```

Comment:
We should establish a better hierarchy for these minor documentation pages. In the grand scheme of OpenEuropa the policy regarding experimental Drupal core modules is very very minor, but here it is placed right on the main page of the initiative, given the same priority as the "OpenEuropa components" and "How to contribute" pages.

This should be something like a 3rd or 4th level page, in a hierarchy similar to "3rd party open source" > "Drupal" > "Policies" > "Drupal core experimental module policy".

Also the position in the docroot should reflect its importance, now it is placed at the root `docs/` folder. This is not a sustainable situation if we are going to document every little decision here in this repo, we're going to end up with hundreds of files. It should be in a place like `docs/3rd-party/drupal/policies/`.

Reply:
This sounds like a completely logical change. How would you go to modify the main Readme file in this case? With a list of elements maybe? I accept suggestions :)